### ISSUE TITLE
website no longer syncs with my account

### ISSUE THREAD

**Issue by TyMick** [HabitRPG/habitica#1368]
The HabitRPG website is no longer syncing with my account.  Both Android clients I've tried, official and third-party, are still showing up correctly, but on the site it appears that my account's been reset.  What's going wrong?

**Comment by lefnire** [comment]
So the mobile apps both sync to eachother, and perform day roll-overs properly, etc? That's good news to me. What specifically isn't syncing on the website, are all edits / scoring / stats / everything not coming in? or is it something specific, like day roll-over not being performed?

We're kinda of in the unfortunate state that the website is in bad shape, and there's not too much we can do about it. I'm currently rewriting from scratch ([see details here](http://habitrpg.tumblr.com/post/59104876969/website-issues-what-were-doing)) and am decently far along. I'll try to fix any current major website bugs I can as I find time, but something like this just yells Racer bug to me, something that could be very difficult / impossible to tackle.

**Comment by TyMick** [comment]
Yeah, both Android apps are syncing with each other just fine, but the website reset me back to Level 1 and deleted my habits/dailies/tasks and put in the default ones.  That reset version of myself, though, is staying consistent, losing HP to the original dailies each day...  It's showing up like this on two different computers.  Using Chrome, if it makes a difference.  And I did just realize that the last thing I did on the website before this happened was check in to the Tavern for the first time, so that may have triggered it somehow.

**Comment by TyMick** [comment]
Oh my goodness, I just found out that the mobile version of the website is displaying my character correctly...  That's weird.  I'm using Chrome for Android.  Maybe I can check out of the Tavern on the mobile site, though...

**Comment by fragmental** [comment]
This may be a silly question, but are you sure that you're logged in correctly, and to the correct website?  try both beta.habitrpg.com and habitrpg.com.  Check settings to make sure that the api and id are correct.

**Comment by TyMick** [comment]
Huh, looks like the desktop website gave me a completely different User ID and API Token.  I'm using the exact same username and password as the mobile apps and mobile website, though, which have the correct User ID and API Token.  Do you know of any way I can change those to the correct ones on the desktop site?

**Comment by TyMick** [comment]
Oh, and I tried the beta website, and the same thing was happening.

**Comment by lefnire** [comment]
Woa... @tw-mick , what are the uuids (leave out api token) for both accounts you're seeing?

**Comment by TyMick** [comment]
Alright, so the correct User id, the one that the mobile apps and mobile website are still giving me, is 0601f1af-b9ce-4a07-9817-e11e0bb1cb44, while the incorrect User ID, which the desktop site is now giving me, is e309cc5c-f369-487e-acfb-2ab16f35db68.  I'm using the exact same username and password in each case, mind you.

**Comment by TyMick** [comment]
Uh oh, I just logged out and back in to the official Android app (which I had done successfully before since the desktop site screwed up), but now it's using the wrong User ID and API Token, same as the desktop site...  Well, at least the mobile site and Habit AndroRPG are still showing up correctly.

**Comment by lefnire** [comment]
ok, will check into this ASAP. sorry for delay, pulling 16h/7's are no fun :/

**Comment by Aubec** [comment]
Hi All.  It's looking like I might have a similar issue.  My iphone app is working properly (like a charm, in fact), however, things I add to the to-do list on the iphone aren't showing up on the website.

**Comment by TyMick** [comment]
Is there a way for me to log in to the website with the API Token and User ID instead of username and password?  I feel like that would fix my problems.

**Comment by Aubec** [comment]
Update, my Iphone Habit now is identical to the new-site Habit (this seems good!).  Old-site Habit isn't updating as I change things.

**Comment by TyMick** [comment]
Wait, what are the old and new sites?  Is this the beta server and the regular server, or is this something else?

**Comment by Aubec** [comment]
My Iphone Habit matches here (the rewritten site): https://habitrpg.herokuapp.com

But not here (the original site): http://www.habitrpg.com/

**Comment by LordGrowl** [comment]
In the most recent update (literally a few minutes ago), the herokuapp and main site are no longer in sync. As soon as the pop up appeared, I saw that I had lost 2 levels, my armour, my shield and gold. I was able to restore these in the main site, but the herokuapp is still 2 levels down, naked and broke.

Edit: Ignore that, all is fine again.

Edit2: It stopped being fine, but lefnire has resolved the issue now (see #1391).
